examples/01-02-ontologies.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Branje ontologij s strežnika\n",
+    "\n",
+    "Primer uporabe API-ja za prenos ontologije iz strežnika, branje in izpisovanje ontologije."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "* Owlready2 * Warning: optimized Cython parser module 'owlready2_optimized' is not available, defaulting to slower Python implementation\n"
```

Comment:
The script is loaded with a Warning, which I do not get with the latest version of owlready2. If owlready2 is missing,  the standard import error appears. Should we provide a script with a warning?

Reply:
Fixed. I didn't install Cython before installing this package and forgot to fix it.